R/utils-data.R

```diff
@@ -39,12 +39,13 @@ get_init <- function(x) {
 #' @section Get a batch of observations:
 #'
 #' By default datasets are iterated by returning each observation/item individually.
-#' Sometimes it's possible to have an optimized implementation to take a batch
+#' Often it's possible to have an optimized implementation to take a batch
 #' of observations (eg, subsetting a tensor by multiple indexes at once is faster than
 #' subsetting once for each index), in this case you can implement a `.getbatch` method
 #' that will be used instead of `.getitem` when getting a batch of observations within
-#' the dataloader. `.getbatch` must work for batches of size larger or equal to 1.
-#'
+#' the dataloader. `.getbatch` must work for batches of size larger or equal to 1. For more
+#' on this see the the vignette on data loading.
```

Comment:
This will make at least the pkgdown website to create a link:

```suggestion
#' on this see the the `vignette("loading-data")`.
```

Up to you if you prefer adding this or not. I think most users will read it in the Rd format.